src/utils/constants.ts

```diff
@@ -112,14 +112,23 @@ export const STELLATE_URLS = {
 };
 
 export const BALANCER_QUERIES = '0xE39B5e3B6D74016b2F6A9673D7d7493B6DF549d5';
-export const BALANCER_POOL_DATA_QUERIES_ADDRESSES: Record<number, Address> = {
-    [ChainId.ARBITRUM_ONE]: '0x7Ba29fE8E83dd6097A7298075C4AFfdBda3121cC',
-    [ChainId.AVALANCHE]: '0x67af5D428d38C5176a286a2371Df691cDD914Fb8',
-    [ChainId.GNOSIS_CHAIN]: '0x3f170631ed9821Ca51A59D996aB095162438DC10',
-    [ChainId.MAINNET]: '0xf5CDdF6feD9C589f1Be04899F48f9738531daD59',
-    [ChainId.OPTIMISM]: '0x6B5dA774890Db7B7b96C6f44e6a4b0F657399E2e',
-    [ChainId.POLYGON]: '0x84813aA3e079A665C0B80F944427eE83cBA63617',
-    [ChainId.ZKEVM]: '0xF24917fB88261a37Cc57F686eBC831a5c0B9fD39',
+export const MULTICALL: Record<number, Address> = {
+    [ChainId.ARBITRUM_ONE]: '0xca11bde05977b3631167028862be2a173976ca11',
+    [ChainId.AVALANCHE]: '0xca11bde05977b3631167028862be2a173976ca11',
+    [ChainId.GNOSIS_CHAIN]: '0xca11bde05977b3631167028862be2a173976ca11',
+    [ChainId.MAINNET]: '0xca11bde05977b3631167028862be2a173976ca11',
+    [ChainId.OPTIMISM]: '0xca11bde05977b3631167028862be2a173976ca11',
+    [ChainId.POLYGON]: '0xca11bde05977b3631167028862be2a173976ca11',
+    [ChainId.ZKEVM]: '0xca11bde05977b3631167028862be2a173976ca11',
+};
```

Comment:
it's always the same, which is a feature, so we can just define it in the multicaller lib.

Reply:
No longer needed now we're using Viems multicall which has config per chain.